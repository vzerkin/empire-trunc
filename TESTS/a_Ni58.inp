 40.001              ;INCIDENT ENERGY (IN LAB)
 58.0  28.0          ;TARGET A, Z
  4.    2.           ;PROJECTILE A, Z
1                    ;NUMBER OF NEUTRONS  TO BE EMITTED
1                    ;NUMBER OF PROTONS  TO BE EMITTED
1                    ;NUMBER OF ALPHAS   TO BE EMITTED
1  2. 1.             ;NUMBER OF L.I. TO BE EMITTED AND ITS A AND Z
IOUT       3.
LEVDEN     0.
NEX       60.
MSD        0.
MSC        0.
ENDF       0.
PCROSS     1.
DEGAS      0.
HMS        0.
DIRECT     3.
GO
-1.
0.002
0.005
0.007
0.01
0.02
0.05
0.07
0.1
0.2
0.5
0.7
1.
1.5
2.
2.5
3.
4.
5.
6.
7.
8.
9.
10.
11.
12.
13.
14.
15.
16.
17.
18.
19.
20.
-1.
NIXSH      1.
FITLEV     2.
LEVDEN     0.
