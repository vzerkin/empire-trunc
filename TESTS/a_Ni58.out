  
 REACTION   2-He-  4 +  28-Ni- 58 INCIDENT ENERGY   40.0     MeV
 COMPOUND NUCLEUS ENERGY   40.785 MeV
 FUSION CROSS SECTION =   1542.74     mb
  30-Zn- 62 production cross section  .216628E-03 mb
    fission  cross section  .00000     mb
 g  emission cross section  228.18     mb
 n  emission cross section  409.58     mb
 p  emission cross section  812.76     mb
 He emission cross section  250.18     mb
 d  emission cross section  97.673     mb
  30-Zn- 61 production cross section  .373687     mb
    fission  cross section  .00000     mb
 g  emission cross section  2.1862     mb
 n  emission cross section  30.962     mb
 p  emission cross section  342.18     mb
 He emission cross section  34.485     mb
 d  emission cross section  1.5839     mb
  29-Cu- 61 production cross section  1.56073     mb
    fission  cross section  .00000     mb
 g  emission cross section  11.626     mb
 n  emission cross section  320.72     mb
 p  emission cross section  368.18     mb
 He emission cross section  114.29     mb
 d  emission cross section  8.0067     mb
  29-Cu- 60 production cross section  218.672     mb
    fission  cross section  .00000     mb
 g  emission cross section  1140.4     mb
 n  emission cross section  24.014     mb
 p  emission cross section  508.91     mb
 He emission cross section  8.8859     mb
 d  emission cross section  .73648E-01 mb
  28-Ni- 58 production cross section  76.6559     mb
    fission  cross section  .00000     mb
 g  emission cross section  141.86     mb
 n  emission cross section  32.305     mb
 p  emission cross section  127.42     mb
 He emission cross section  13.336     mb
 d  emission cross section  .53068     mb
  28-Ni- 57 production cross section  40.5635     mb
    fission  cross section  .00000     mb
 g  emission cross section  213.04     mb
 n  emission cross section  1.0926     mb
 p  emission cross section  25.102     mb
 He emission cross section  .31630E-01 mb
 d  emission cross section  .20640E-05 mb
  27-Co- 57 production cross section  164.733     mb
    fission  cross section  .00000     mb
 g  emission cross section  838.76     mb
 n  emission cross section  19.575     mb
 p  emission cross section  56.733     mb
 He emission cross section  .67180     mb
 d  emission cross section  .43280E-04 mb
  27-Co- 56 production cross section  54.0531     mb
    fission  cross section  .00000     mb
 g  emission cross section  81.637     mb
 n  emission cross section  .28132E-03 mb
 p  emission cross section  .38593E-01 mb
 He emission cross section  .29152E-09 mb
 d  emission cross section  .00000     mb
  29-Cu- 59 production cross section  22.7127     mb
    fission  cross section  .00000     mb
 g  emission cross section  54.059     mb
 n  emission cross section  .00000     mb
 p  emission cross section  2.8847     mb
 He emission cross section  .52490E-03 mb
 d  emission cross section  .00000     mb
  28-Ni- 59 production cross section  514.601     mb
    fission  cross section  .00000     mb
 g  emission cross section  1581.6     mb
 n  emission cross section  1.7320     mb
 p  emission cross section  .55926     mb
 He emission cross section  .46679E-02 mb
 d  emission cross section  .00000     mb
  27-Co- 55 production cross section  .808277E-03 mb
    fission  cross section  .00000     mb
 g  emission cross section  .10895E-07 mb
 n  emission cross section  .00000     mb
 p  emission cross section  .00000     mb
 He emission cross section  .00000     mb
 d  emission cross section  .00000     mb
  26-Fe- 55 production cross section  .433039E-01 mb
    fission  cross section  .00000     mb
 g  emission cross section  .14967E-01 mb
 n  emission cross section  .00000     mb
 p  emission cross section  .00000     mb
 He emission cross section  .00000     mb
 d  emission cross section  .00000     mb
  26-Fe- 54 production cross section  11.6622     mb
    fission  cross section  .00000     mb
 g  emission cross section  61.008     mb
 n  emission cross section  .29678E-01 mb
 p  emission cross section  1.6433     mb
 He emission cross section  .11187E-02 mb
 d  emission cross section  .00000     mb
  
 Tot. fission cross section    .0000     mb
  
